package.json

```
@@ -2,7 +2,13 @@
   "name": "eslint-plugin-camunda-licensed",
   "version": "0.4.6",
   "description": "Shared lint rules for Camunda licensed OSS projects",
-  "main": "index.js",
+  "type": "module",
+  "exports": {
+    ".": "./index.js",
+    "./configs/apache.js": "./configs/apache.js",
+    "./configs/commercial.js": "./configs/commercial.js",
+    "./configs/mit.js": "./configs/mit.js"
```

```suggestion
    "./configs/*.js": "./configs/*.js"
```

Cf. https://github.com/bpmn-io/eslint-plugin-bpmn-io/blob/main/package.json#L8.